conform-pr/main_test.go

```diff
@@ -133,35 +133,55 @@ func TestCheckTitle(t *testing.T) {
 		name        string
 		title       string
 		expectedErr error
-	}{{
-		name:        "pull_request/title_without_dot",
-		title:       "Test the title without a dot",
-		expectedErr: nil,
-	}, {
-		name:        "pull_request/title_with_a_digit",
-		title:       "Test the title without a digit 1",
-		expectedErr: nil,
-	}, {
-		name:        "pull_request/title_with_dot",
-		title:       "I'm a title with a dot.",
-		expectedErr: errors.New("PR title must end with a latin letter or digit."),
-	}, {
-		name:        "pull_request/title_with_whitespace",
-		title:       "I'm a title with a whitespace ",
-		expectedErr: errors.New("PR title must end with a latin letter or digit."),
-	}, {
-		name:  "pull_request/title_without_imperative_verb",
-		title: "I'm a title without an imperative verb at the beginning",
-		// TODO expectedErr: errors.New("PR title must start with an imperative verb."),
-	}, {
-		name:        "pull_request/title_with_backticks",
-		title:       "Test the title I'm a title with a `backticks`",
-		expectedErr: nil,
-	}, {
-		name:        "pull_request/title_without_uppercase",
-		title:       "test the title that does not start with an uppercase`",
-		expectedErr: errors.New("PR title must start with an uppercase letter."),
-	}}
+	}{
+		{
```

Comment:
```suggestion
	}{{
```

That will make `go fmt` happy